setup.cfg

```diff
@@ -10,10 +10,11 @@ markers =
     needs_connection: requires an internet connection (deselect with '-m "not needs_connection"')
 
 [flake8]
-ignore = E, W, C901, radis/io/hapi.py, */__init__.py
+ignore = C901
 max-line-length = 89
 max-complexity = 18
-select = B,C,E,F,W,T4,B9
+select = B,C,F,T4,B9
+per-file-ignores = __init__.py:F401,F403
```

Comment:
@suzil  the `radis/io/hapi.py` does not look not ignored here (it's the Python interface from the HITRAN group, we include it in RADIS for comparison in the tests as they didn't make it a library installable from pip. But it does not comply with flake so should be ignored. ) 
You tests passed, though, so it is probably ignored anyway ?

Reply:
I thought it would be OK to remove it as ignored since there are no current flake8 violations?